caluma/core/mutation.py

```diff
@@ -243,4 +243,10 @@ def get_object(cls, root, info, queryset, **input):
                 "No %s matches the given query." % queryset.model._meta.object_name
             )
 
+        if instance is None and "create" not in cls._meta.model_operations:
+            raise exceptions.ValidationError("Create model operation not allowed.")
+
+        if instance is not None and "update" not in cls._meta.model_operations:
+            raise exceptions.ValidationError("Update model operation not allowed.")
```

Comment:
I'm not entirely convinced that this is the right place for such validations. However I also don't know any better place off the top of my head